content/6_Advanced/DP_More.mdx

```diff
@@ -54,7 +70,13 @@ Queue with 2 stacks is used to remove a factor of $\mathcal{O}(\log N)$ in [USAC
 
 </Optional> -->
 
-### "Connected Component" DP Problems
+### Connected-Component DP Problems
+
+<IncompleteSection>
+
+Make Permutations II a focus problem, move analysis to module.
+
+</IncompleteSection>
```

Comment:
do this yourself for this to be counted as an actual contribution

Reply:
what

Reply:
so the work I've done here isn't a contribution?

Reply:
I'm not getting paid to do this nor do I have any obligation to do so, so any work is a contribution?

Reply:
anyways I corrected myself earlier, so let’s just stop this